crates/database/src/orders.rs

```diff
@@ -1616,6 +1675,17 @@ mod tests {
         assert!(!get_full_order(&mut db).await.unwrap().presignature_pending);
     }
 
+    #[tokio::test]
+    async fn get_orders_with_quote() {
+        let mut db = PgConnection::connect("postgresql://").await.unwrap();
+        let mut db = db.begin().await.unwrap();
+        assert!(
+            user_orders_with_quote(&mut db, 0, &Default::default())
```

Comment:
This was using the orignal OPEN_ORDERS but had no tests whatsoever, so I wanted to at least make sure the query is correctly formatted.